assets/src/util/operatorFormatting.tsx

```diff
@@ -0,0 +1,37 @@
+import { joinTruthy } from "../helpers/dom"
+import { Vehicle } from "../realtime"
+
+export const DefaultFallbackString = "Not Available"
+
+type OptionParameters = {
+  fallbackText?: string
+}
+
+export const formatOperatorName = (
+  operatorFirstName: string | null,
+  operatorLastName: string | null,
+  operatorId: string | null,
+  _options?: OptionParameters
```

Comment:
**nitpick:** we usually use underscore-prefixed variables to denote arguments that aren't read (as per Elixir convention), but `_options` is referenced in the function body.

Reply:
I missed this while ensuring tests worked, thank you! 🙇🏻‍♀️